dandi/models.py

```diff
@@ -888,19 +956,23 @@ class BareAssetMeta(CommonModel):
 class AssetMeta(BareAssetMeta, Identifiable):
     """Metadata used to describe an asset on the server."""
 
-    identifier: UUID4 = Field(readOnly=True, nskey="schema")
-
+    id: Optional[str] = Field(readOnly=True, description="Dandi asset id")
```

Comment:
```suggestion
    id: Optional[str] = Field(readOnly=True, description="Uniform resource identifier")
```
to stay inline with how we have it for DandisetMeta? 

initial attention was triggered by use of `"Dandi ..."`.  We do use `Dandi` in class names, but shouldn't it be `DANDI` if referring in text, and wouldn't everything be `Dandi` here?